src/vm/eetwain.cpp

```diff
@@ -5729,6 +5729,7 @@ BOOL EECodeManager::IsInFilter(GCInfoToken gcInfoToken,
 }
 
 
+#if defined(_TARGET_X86_) && !defined(WIN64EXCEPTIONS)
```

Comment:
As Esp is x86-specific, and thus I used _TARGET_X86_ && !WIN64EXCEPTIONS here.

Is it better to revise this as follows?
```
#ifndef WIN64EXCEPTIONS
#ifdef _TARGET_X86_
~~~
#else
#error "Unsupported"
#endif
#else
~~~
#endif
```